inttest/hacontrolplane/hacontrolplane_test.go

```diff
@@ -110,6 +110,15 @@ func (s *HAControlplaneSuite) TestDeregistration() {
 	s.Equal(membersFromMain, membersFromJoined,
 		"etcd cluster members list must be the same across all nodes")
 	s.Len(membersFromJoined, 2, "etcd cluster must have exactly 2 members")
+
+	// Restart the second controller with a token to see it comes up
+	// It should just ignore the token as there's CA etc already in place
+	sshC1, err := s.SSH("controller1")
+	s.Require().NoError(err)
+	_, _ = sshC1.ExecWithOutput("kill $(pidof mke)")
```

Comment:
Do we have to use return values here?

Reply:
not really. if `kill $(pidof mke)` would fail for some reason it will make also the next steps fail.

Reply:
mke -> k0s